src/Queue/Jobs/RabbitMQJob.php

```diff
@@ -75,15 +75,14 @@ public function attempts(): int
         $headers = Arr::get($this->message->get_properties(), 'application_headers');
 
         if (! $headers) {
-            return 0;
+            return 1;
```

Comment:
By default, all Laravel queue drivers return 0 for first attempt.

Reply:
This is true when pushing a job for the first time to the queue. 
When a cosumer/worker gets the message for the first time attempts() must return the current try. 

Sea also the RedisJob of the framework:

[Link](https://github.com/jimrubenstein/laravel-framework/blob/f7ba54c132319631163ac0c98101da0a3b16c3ca/src/Illuminate/Queue/Jobs/RedisJob.php#L108)